src/components/RepoItem/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { StarFilled } from '@ant-design/icons';
+import React, { useCallback } from 'react';
+import styled from 'styled-components';
+
+interface Props {
+  html_url: string;
+  full_name: string;
+  stargazers_count: number;
+}
+
+export const RepoItem: React.FC<Props> = (props) => {
+  const { html_url: htmlUrl, full_name: fullName, stargazers_count: starCount } = props;
```

Comment:
처음 데이터를 내려줄때 바꿔서 내려 주는건 어떨까요?
html_url -> htmlUrl

Reply:
```suggestion
  this.data = response.data.map((el: ResponseType) => ({
        htmlUrl: el.html_url,
        fullName: el.full_name,
        starCounts: el.stargazers_count
      }));
```

데이터 fetch 후 이런식으로 수정했습니다 감사합니다!!